integration_test/install_test.go

```diff
@@ -58,7 +58,7 @@ func TestKrewInstall_ManifestAndArchive(t *testing.T) {
 		"--manifest", filepath.Join("testdata", fooPlugin+constants.ManifestExtension),
 		"--archive", filepath.Join("testdata", fooPlugin+".tar.gz")).
 		RunOrFail()
-	test.AssertExecutableInPATH("kubectl-" + strings.ReplaceAll(fooPlugin, "-", "_"))
```

Comment:
we can bump to 1.12 if necessary.

Reply:
That's a good idea anyway. But I would not do it for such a small inconvenience.

Reply:
Renaming from `test-foo` to `foo` even made this `ReplaceAll` obsolete.